sdk/servicebus/Azure.Messaging.ServiceBus/src/ServiceBusMessage.cs

```diff
@@ -34,13 +32,34 @@ public ServiceBusMessage()
         {
         }
 
+        /// <summary>
+        /// Creates a new message from the specified string, using UTF-8 encoding.
+        /// </summary>
+        /// <param name="body">The payload of the message as a string.</param>
+        public ServiceBusMessage(string body) :
```

Comment:
Should we have an overload that takes an instance of `BinaryData` as well?

Reply:
Actually, you're right. Might as well add it now.